UnityProject/Assets/Scripts/UI/ControlChat.cs

```diff
@@ -73,6 +73,7 @@ public class ControlChat : MonoBehaviour
 	/// Are the channel toggles on show?
 	/// </summary>
 	private bool showChannels = false;
+	private bool debounceCloseChat = false;
```

Comment:
This debounce var and the logic is unnecessary, the behaviour seemed the same with and without it